.github/workflows/publish-cli-bins.yaml

```diff
@@ -0,0 +1,53 @@
+name: "Publish GitHub CLI Binaries"
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    strategy: 
+      matrix:
+        os: [linux, darwin]
+        arch: [amd64, arm64]
+    env:
+      NAME: flightctl-${{ matrix.os }}-${{ matrix.arch }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v4
+
+      - name: "Build"
```

Comment:
we should invoke the binaries to ensure they are sane. you can use darwin runners
```
jobs:
  build-mac:
    runs-on: macos-12
```

Reply:
Ok, I added a test for linux/amd64 and darwin/arm64